### PR TITLE
updates rules README by removing some ssl-regular mentions

### PR DESCRIPTION
**Pull request by PedroPizarro**

### CODE REVIEW

File: README.md
Comment (ersaraujo) [inline README.md:1]: Why don't we maintain the automatic generation of files? I can help with that.

**Review by ersaraujo — CHANGES_REQUESTED** [review]

File: README.md
Comment (PedroPizarro) [reply README.md:1]: We can keep it, we only must update and maintain it.
If you say that you can help with that, given a clear working process of CI, we as TC can maintain it (if no additional cost is involved, IMO).

Comment (JoaoVictor0508) [reply README.md:1]: I agree that we can keep it. Maybe we could create a basic documentation for future TCs.

Comment (PedroPizarro) [reply README.md:1]: Great,
@JoaoVictor0508  @ersaraujo do you prefer to close this PR and create another one with the updated CI files/scripts, or just commit directly into this PR?

Comment (joaocarloscampi) [reply README.md:1]: I agree that we can keep it too. This will help teams access all of the rules faster and more easily.

Comment (PedroPizarro) [reply README.md:1]: @ersaraujo  @JoaoVictor0508  Should we close this PR and create another one, or you are going to commit directly into this PR?

Comment (JoaoVictor0508) [reply README.md:1]: We can keep this PR, or can open another one to keep it organized and mention this PR. I prefer the second. What do you think?

**Pull request closed without merging**